packages/xo-web/src/xo-app/pool/tab-network.js

```diff
@@ -101,6 +101,45 @@ class DefaultPif extends BaseComponent {
   }
 }
 
+class Nbd extends Component {
+  NBD_FILTER_OPTIONS = [
+    {
+      label: 'noNbdConnection',
+      value: false,
+    },
+    {
+      label: 'nbdConnection',
+      value: true,
+    },
+  ]
+  INSECURE_OPTION = [
+    {
+      label: 'insecureNbdConnection',
+      value: 'insecure_nbd',
+      disabled: true,
+    },
+  ]
+
+  _getOptionRenderer = ({ label }) => <span>{_(label)}</span>
+
+  _editNbdConnection = value => {
+    editNetwork(this.props.network, { nbd: value.value })
+  }
+
+  render() {
+    const { network } = this.props
+
+    return (
+      <Select
+        onChange={this._editNbdConnection}
+        optionRenderer={this._getOptionRenderer}
+        options={network.insecureNbd ? [...this.NBD_FILTER_OPTIONS, ...this.INSECURE_OPTION] : this.NBD_FILTER_OPTIONS}
```

Comment:
So we don't even show the insecure option when it's not used, is that correct? Could you add a comment in the code to quickly explain that and why we decide to do that since it's not very common?

Reply:
Yes, that's correct. Since we don't allow the user to activate this option through XO, we'll show it only when it is already activate so the users that don't know about it won't see it.

I'll put a comment.